automatic/git.portable/tools/chocolateyUninstall.ps1

```diff
@@ -8,4 +8,4 @@ Remove-Item $installLocation -Recurse -Force -ea 0
 $newPath = $Env:Path.Replace(";$installLocation", '')
 if ($newPath -eq $Env:PATH) { return }
 Write-Host "Removing Git from system PATH"
-[System.Environment]::SetEnvironmentVariable('PATH', $newPath, 'Machine')
\ No newline at end of file
+[System.Environment]::SetEnvironmentVariable('PATH', $newPath, 'Machine')
```

Comment:
@majkinetor that may be, but portable packages are supposed to work without needing administrative privileges.